.gitignore

```diff
@@ -31,3 +31,5 @@ integration/kubeconfig-*
 # Generated header file
 .license-header
 
+# Go packages eksctl depends on, for the build cache
+go-deps.txt
```

Comment:
Would you mind calling it `.go-deps` or `.go-dependencies` or something? Dot-TXT is so Windows-y... ;)

Reply:
I would rather have an explicit file rather than a hidden one. But sure ....

Reply:
This is no longer needed (we don't generate a deps file anymore)

Reply:
Very good, thanks!